SKILL.tsv

```diff
@@ -3719,45 +3719,45 @@ SKILL_20150414_003718	Order summoned demon to move and attack in target area.
 SKILL_20150414_003719	Ride on top of the devil summoned with Grimore and control it.
 SKILL_20150414_003720	Mutate the devil summoned with Grimore to the Devil in the sub card equippped on Grimore while maintaining its previous abilities.
 SKILL_20150414_003721	Order summoned devil to wait and guard target area.
-SKILL_20150414_003722	{#993399}{ol}[Magic] - [Dark]{/}{/}{nl}Summon familiar spirit. Spirit rush to enemy when you attack or be attacked then explodes itself.
-SKILL_20150414_003723	Summon fire devil Salamion. It follows its master and attacks enemy.
+SKILL_20150414_003722	{#993399}{ol}[Magic] - [Dark]{/}{/}{nl}Summon a familiar spirit. The spirit rushes to a target that you attack or that attacks you, and then explodes itself.
+SKILL_20150414_003723	Summon a fire devil Salamion. It follows its master and attacks enemies.
 SKILL_20150414_003724	{memo X}Salamion Level #{CaptionRatio}#
-SKILL_20150414_003725	Summon Servant. Servant follows its master and gives good effects.
+SKILL_20150414_003725	Summon Servant. Servant follows its master and gives beneficial effects.
```

Comment:
Summons a servant. The servant will grant beneficial effects while following its master.